src/util.ts

```diff
@@ -18,6 +18,17 @@ export function die(message: string): never {
 	process.exit(1);
 }
 
+export function getIndentation(text: string): number {
+	const lines = text.split(/\r?\n/);
+	if (lines.length >= 2) {
```

Comment:
maybe find the first line with "{" as content ... and then check line afterwards? This could prevebnt issues with empty lines on file start (unusual but edge case?)

WDYT?